bower.json

```diff
@@ -18,7 +18,7 @@
     "patternfly": "3.21.0",
     "jquery": "2.1.4",
     "lodash": "3.10.1",
-    "origin-web-common": "^0.0.10"
+    "origin-web-common": "~0.x"
```

Comment:
Based on the comment of the PR yes, but maybe we should just pickup any minor version changes.